arch/arm/src/stm32h7/Kconfig

```diff
@@ -616,7 +616,7 @@ config STM32H7_HAVE_GPIOF
 
 config STM32H7_HAVE_GPIOG
 	bool
-	default n
+	default y
```

Comment:
none STM32H7_HAVE_GPIOx should be enabled by default, this symbol is enabled conditionally by the MCU family